acstools/acsphotcte.py

```diff
@@ -0,0 +1,289 @@
+
+            # Check if the input is a python float, not an np.float32
+            # Handles the case of str inputs as well
+            elif not isinstance(inputs[key], float):
+                try:
+                    inputs[key] = float(inputs[key])
+                except TypeError as e:
+                    # TypeError during the conversion, print the info and
+                    # return the key it failed on
+                    LOG.info(e)
+                    return key
+                else:
+                    if key == 'radius' and inputs[key] not in [3., 5.]:
+                        LOG.info('Submitted radii not supported...')
+                        return key
+        return inputs
+
+    def _compute_corrections(
+            self,
+            radius=None,
+            ytransfers=None,
+            mjd=None,
+            local_skys=None,
+            fluxes=None,
+            api_id=None,
+            api_key=None,
+    ):
+        """Compute the photometric CTE corrections
+
+        Parameters
+        ----------
+        radius : {3, 5}
+            Aperture radius used to extract photometry
+
+        ytransfers : float, list
+            Number of y transfers. For chip 2 on WFC this is just the Y value,
+            for chip 1 it is 2048 - Y value.
+
+        mjd : float
+            MJD date of the observation
+
+        local_sky : float, list
+            Local sky background for each source
+
+        fluxes : float, list
+            Computed source flux, in electrons, for 3 or 5 pixel apertures.
+
+        Returns
+        -------
+
+        """
+
+        api_credentials = {
+            'x-api-id': api_id,
+            'x-api-key': api_key
+        }
+        inputs = {
+            'radius': radius,
+            'ytransfers': ytransfers,
+            'mjd': mjd,
+            'local_skys': local_skys,
+            'fluxes': fluxes
+        }
+        inputs = self._check_inputs(**inputs)
+        if not isinstance(inputs, dict):
+            LOG.info(f'Did you forget to supply the {inputs}?')
+            return
+        content = self._query(inputs, api_credentials)
+
+        # Update the attribute and return the value
+        self.cte_corrections = np.array(content['deltamag'])
+        return self.cte_corrections
+
+    def correct_photometry(
+            self,
+            radius=None,
+            ytransfers=None,
+            mjd=None,
+            local_skys=None,
+            fluxes=None,
+            api_id=None,
+            api_key=None
+    ):
+        """Get the CTE corrected FLT photometry
+
+        Parameters
+        ----------
+        radius : {3, 5}
+            Aperture radius used to extract photometry
+
+        ytransfers : float, list
+            Number of y transfers. For chip 2 on WFC this is just the Y value,
+            for chip 1 it is 2048 - Y value.
+
+        mjd : float
+            MJD date of the observation
```

Comment:
Not everyone think in MJD.  I wonder if it is better to support `YYYY-MM-DD` but that discussion is out of scope here.
```suggestion
            MJD date of the observation.
```